code/game/objects/items/storage/pouch.dm

```diff
@@ -812,6 +812,12 @@
 		/obj/item/roller = list(SKILL_MEDICAL, SKILL_MEDICAL_MEDIC),
 		/obj/item/bodybag = list(SKILL_MEDICAL, SKILL_MEDICAL_MEDIC),
 		/obj/item/reagent_container/blood = list(SKILL_MEDICAL, SKILL_MEDICAL_MEDIC),
```

Comment:
This line here should account for all the bloodbags, it fits now in game. I fixed it before merge.

Reply:
I shall beat the folk who told me it was still borken with rusty spoon. Cut the medkit part left the vendor blood bag part rather than opening a separate PR